voltaire_bundler/utils/cache.py

```
@@ -0,0 +1,564 @@
+                "--disable_persistent_cache to silence, or "
+                "--clear_cache to wipe a corrupted DB.",
+                self.name, type(exc).__name__, exc,
+            )
+            # _open_db rolls back its own connections on failure, but a
+            # warmup-time error leaves them open — close them here so the
+            # memory-only path doesn't leak file handles.
+            if self._write_conn is not None or self._read_conn is not None:
+                try:
+                    await asyncio.to_thread(self._close_conns)
+                except Exception:
+                    pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)